docs/ocs-on-a-disconnected-environment.md

```diff
@@ -53,129 +53,150 @@ you should get something similar to this:
-registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:203417dc512f0e3583be06db7f93a3729fef08d8e939ec1fc828ef914d1d8995=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/openshift4/ose-csi-external-provisioner-rhel7
-registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ad5dc22e6115adc0d875f6d2eb44b2ba594d07330a600e67bf3de49e02cab5b0=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/rhscl/mongodb-36-rhel7
-```
+below are examples of a mapping file for **OCS-4.3.0**  and **OCS-4.4.0** (other versions will require different images. go over the CSV and make sure that all of the used images (can be grepped with `sha256`) are mapped in the file)  
+save the content below to `mapping.txt`  
+
+  mapping.txt for OCS-4.3.0:
+  ```
+  registry.redhat.io/ocs4/cephcsi-rhel8@sha256:9aa61eec1d23169521f83b4664115d9aa69710d69d7f2f4a939ec753d15c0400=<your_registry>/ocs4/cephcsi-rhel8
+  registry.redhat.io/ocs4/mcg-core-rhel8@sha256:74d03cc253774bd817750ba0fe3cfec125282f8549343067610799f2eee5ea80=<your_registry>/ocs4/mcg-core-rhel8
+  registry.redhat.io/rhceph/rhceph-4-rhel8@sha256:9e521d33c1b3c7f5899a8a5f36eee423b8003827b7d12d780a58a701d0a64f0d=<your_registry>/rhceph/rhceph-4-rhel8
+  registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:b40c8e8392e432f49c4abf6b5248fb6a349ab9dff49c7f9467df3fc974937a6d=<your_registry>/openshift4/ose-csi-driver-registrar
+  registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:faeb96365fafba1255ca3b9ad7e3f6ddd8791588b9ff283a7acffd7d5c89fe09=<your_registry>/openshift4/ose-csi-external-attacher
+  registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:0570eacd3cb8b5d60fd3ecf4c80860280725c81d83bc2a04326d3393c6dcb2be=<your_registry>/openshift4/ose-csi-external-provisioner-rhel7
+  registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:e7302652fe3f698f8211742d08b2dcea9d77925de458eb30c20789e12ee7ae33=<your_registry>/openshift4/ose-csi-external-resizer-rhel7
+  registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:254c9046eaf61c451b40bac3f897c40bc59f187e5313b9799fbcacf17398a191=<your_registry>/rhscl/mongodb-36-rhel7
+  quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=<your_registry>/noobaa/pause
+  registry.redhat.io/ocs4/ocs-must-gather-rhel8@sha256:a512e88410cadd6e4b286aca63b8917389dd73e8c202189563055c37f4c50fef=<your_registry>/ocs4/ocs-must-gather-rhel8
+  quay.io/noobaa/lib-bucket-catalog@sha256:b9c9431735cf34017b4ecb2b334c3956b2a2322ce31ac88b29b1e4faf6c7fe7d=<your_registry>/noobaa/lib-bucket-catalog
+  ```
+
+  in OCS-4.4 the relatedImages do appear in the CSV, but we still need to fix the missing pause image for lib-bucket-provisioner.  
+  Also the `ose-csi-external-resizer` image is missing in the relatedImage (Although it is not yet used by OCS we'll mirror it anyway)
+
+  mapping.txt for OCS-4.4.0:
+  ```
```

Comment:
Also missing look like this image from testing in air-gap disconnected env using relatedimages to populate registry pod for OCS 4.4 install.

```
registry.redhat.io/ocs4/ocs-rhel8-operator@sha256:78b97049b194ebf4f72e29ac83b0d4f8aaa5659970691ff459bf19cfd661e93a
```

Reply:
`relatedImages` is used for mirroring operands images that are not referenced by the CSV. ocs operator image is already referenced in the CSV and should be mirrored by `oc adm catalog mirror` regardless of the relatedImages